src/app/Navigation/Navigation.js

```diff
@@ -1,23 +1,26 @@
 import styled from 'styled-components/macro'
-import PropTypes from 'prop-types'
 import { NavLink } from 'react-router-dom'
 import {PlusIcon, HomeIcon} from '../Icons/Icons'
+import removeLocally from '../../lib/removeLocally'
 
-Navigation.propTypes = {
-    handleClick: PropTypes.func.isRequired
-}
-
-export default function Navigation({handleClick}) {
+export default function Navigation() {
     return (
             <NavBar>
-                <NavLinkStyled exact to="/" onClick={handleClick} data-testid="button-home-page">
+                <NavLinkStyled exact to="/" onClick={emptyLocalStorage} data-testid="button-home-page">
                     <HomeIcon />
                 </NavLinkStyled>
-                <NavLinkStyled to="/create" onClick={handleClick} data-testid="button-form-page">
+                <NavLinkStyled to="/create" onClick={emptyLocalStorage} data-testid="button-form-page">
                     <PlusIcon />
                 </NavLinkStyled>
             </NavBar>
     )
+
+    function emptyLocalStorage() {
+        removeLocally('inputsKeyInfosCreate')
+        removeLocally('inputsScoresCreate')
+        removeLocally('inputsKeyInfosEdit')
+        removeLocally('inputsScoresEdit')
+    }
 }
```

Comment:
Just out of curiosity, why is it necessary?

Reply:
When a user is working with the form in either way (creating or editing) and clicks on a navigation button, the creating or editing process will be aborted. But due to the local storage, the data will be still saved. In order to have a fresh / new form with no filled inputs or the right data, it is necessary to clean the local storage. And that was the way how it worked for now.